integration_tests/e2e/activities/attendance/multipleActivities.cy.ts

```diff
@@ -210,4 +211,64 @@ context('Record attendance', () => {
       'View or Edit',
     )
   })
+
+  it('Mark attendances for single activity when not in standalone mode', () => {
+    const indexPage = Page.verifyOnPage(IndexPage)
+    indexPage.activitiesCard().click()
+
+    const activitiesIndexPage = Page.verifyOnPage(ActivitiesIndexPage)
+    activitiesIndexPage.recordAttendanceCard().click()
+
+    const recordAttendancePage = Page.verifyOnPage(AttendanceDashboardPage)
+    recordAttendancePage.recordAttendanceCard().click()
+
+    const selectPeriodPage = Page.verifyOnPage(SelectPeriodPage)
+    selectPeriodPage.enterDate(new Date(todayStr))
+    selectPeriodPage.selectAM()
+    selectPeriodPage.continue()
+
+    const activitiesPage = Page.verifyOnPage(ActivitiesPage)
+    activitiesPage.containsActivities('English level 1', 'English level 2', 'Football', 'Maths level 1')
+    activitiesPage.selectActivitiesWithNames('English level 1')
+    activitiesPage.getButton('Record or edit attendance').click()
+
+    const attendanceListPage = Page.verifyOnPage(SingleAttendanceListPage)
+    attendanceListPage.checkAttendanceStatus('Andy, Booking', 'Attended')
+    attendanceListPage.checkAttendanceStatus('Andy, Booking', 'Pay')
+    attendanceListPage
+      .backLink()
+      .contains(`Go back to activities for ${formatDate(today, 'EEEE, d MMMM yyyy')} - AM`)
+      .click()
+
+    Page.verifyOnPage(ActivitiesPage)
+  })
+
+  it('Mark attendances for single activity in standalone mode', () => {
```

Comment:
Ensure that any previous stored session is deleted if attendance list view is called from outside the attendance journey - indicated by no back back link